### PR TITLE
[TEVA-1295] update alert email copy

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-1295

'You can manage your alert here.' text is not added as part of this task but will be added when dashboard is implemented

### CODE REVIEW

File: app/models/subscription.rb [hunk 1]
```diff
@@ -6,6 +6,8 @@ class Subscription < ApplicationRecord
 
   include Auditor::Model
 
+  attr_reader :subject
```
Comment (david-mears-dfe) [inline app/models/subscription.rb:9]: I don't understand how this works - Subscription objects don't have a `subject` column, and I can't see a `subject` method here either.

File: app/views/alert_mailer/daily_alert.text.haml [hunk 2]
```diff
@@ -15,7 +14,13 @@
   \---
   \
 \
-Visit #{notify_link('https://teaching-vacancies.service.gov.uk', t('app.title'))} - the free service for schools in England to list teaching roles and for jobseekers to find them.
+\# #This job alert
```
Comment (david-mears-dfe) [inline app/views/alert_mailer/daily_alert.text.haml:17]: I guess this text 'This job alert' (and the below 'Visit') should be in the en.yml.

File: app/models/subscription.rb (hunk 1 above)
Comment (alexbowen) [reply app/models/subscription.rb:9]: it is needed so that the test can access subject

Comment (david-mears-dfe) [reply app/models/subscription.rb:9]: What is the test accessing? A nil value?

File: spec/mailers/alert_mailer_spec.rb [hunk 3]
```diff
@@ -66,14 +65,16 @@
...
-          ),
+          I18n.t('job_alerts.alert.email.subject'),
         )
         expect(mail.to).to eq([subscription.email])
 
         expect(body).to match(/\[#{first_vacancy_presenter.job_title}\]/)
+        expect(body).to match(
+          /2 new jobs matching your search criteria were listed in the last day./,
+        )
+        expect(body).to match(/Subject: #{subscription.subject}/)
```
Comment (david-mears-dfe) [inline spec/mailers/alert_mailer_spec.rb:76]: I think `subscription.subject` returns `nil` - correct?

File: app/models/subscription.rb (hunk 1 above)
Comment (david-mears-dfe) [reply app/models/subscription.rb:9]: Maybe you meant `reference` instead of `subject`?

File: spec/mailers/alert_mailer_spec.rb (hunk 3 above)
Comment (alexbowen) [reply spec/mailers/alert_mailer_spec.rb:76]: ah yes i think you are right, so the test is a false positive

**Review by david-mears-dfe — APPROVED** [review]

File: spec/mailers/alert_mailer_spec.rb [hunk 4]
```diff
@@ -8,7 +8,7 @@
   let(:body) { mail.body.raw_source }
   let(:subscription) do
     subscription = create(:daily_subscription, email: 'an@email.com',
-                                               reference: 'a-reference',
+                                               frequency: 'daily',
                                                search_criteria: {
                                                  subject: 'English',
```
Comment (david-mears-dfe) [inline spec/mailers/alert_mailer_spec.rb:13]: In Cesi's PR, he rightly corrects 'subject' to 'keyword' here as we don't filter by subject (we used to, a year ago).

File: spec/mailers/alert_mailer_spec.rb [hunk 5]
```diff
@@ -37,18 +37,17 @@
...
         expect(mail.to).to eq([subscription.email])
 
-        expect(body).to match(/# #{I18n.t('app.title')}/)
+        expect(body).to match(/# #{I18n.t('job_alerts.alert.email.daily.summary', count: 1)}/)
         expect(body).to match(
-          /A new job matching your search criteria &#39;#{subscription.reference}&#39; was posted yesterday/,
+          /A new job matching your search criteria was listed in the last day./,
         )
         expect(body).to match(/---/)
+        expect(body).to match(/Subject: English/)
```
Comment (david-mears-dfe) [inline spec/mailers/alert_mailer_spec.rb:49]: As above - Keyword not Subject

File: spec/mailers/alert_mailer_spec.rb [hunk 6]
```diff
@@ -66,14 +65,16 @@
...
-          ),
+          I18n.t('job_alerts.alert.email.subject'),
         )
         expect(mail.to).to eq([subscription.email])
 
         expect(body).to match(/\[#{first_vacancy_presenter.job_title}\]/)
+        expect(body).to match(
+          /2 new jobs matching your search criteria were listed in the last day./,
+        )
+        expect(body).to match(/Subject: English/)
```
Comment (david-mears-dfe) [inline spec/mailers/alert_mailer_spec.rb:76]: As above - Keyword not Subject

**Review by david-mears-dfe — CHANGES_REQUESTED** [review]
Sorry, just spotted something - see comments

**Comment by alexbowen** [comment]
@david-mears-dfe yes i saw this change in cesi's. i need to close this PR and create another as the environment deployment is messed up for this one

**Pull request closed without merging**